src/atomate2/vasp/files.py

```diff
@@ -192,7 +192,7 @@ def write_vasp_input_set(
     )
 
     if apply_incar_updates:
-        vis.incar.update(SETTINGS.VASP_INCAR_UPDATES)
+        vis["INCAR"].update(SETTINGS.VASP_INCAR_UPDATES)
```

Comment:
might make sense to add

```
@property
def incar(self) -> Incar:
    """INCAR object."""
    return self["INCAR"]
```

and co for POTCAR, POSCAR, ... to `pymatgen` `DictSet` so we can keep `vis.incar` here

Reply:
Done on the pmg side